src/Components/Components/src/ICascadingValueComponent.cs

```diff
@@ -11,9 +11,9 @@ internal interface ICascadingValueComponent
         // This interface exists only so that CascadingParameterState has a way
         // to work with all CascadingValue<T> types regardless of T.
 
-        bool CanSupplyValue(Type valueType, string valueName);
+        bool CanSupplyValue(Type valueType, string? valueName);
```

Comment:
Why declare this as nullable?

Reply:
I think that refers to the Name property of the cascading parameter, which can be null.